### PR TITLE
Add support for python_callable_file for PythonSensor

### PR DESCRIPTION
**Pull request by arnekaiser-otto**

### CODE REVIEW

File: dagfactory/dagbuilder.py [hunk 1]
```diff
@@ -194,14 +195,14 @@ def make_task(operator: str, task_params: Dict[str, Any]) -> BaseOperator:
...
             raise Exception(f"Failed to import operator: {operator}") from err
         try:
-            if operator_obj in [PythonOperator, BranchPythonOperator]:
+            if operator_obj in [PythonOperator, BranchPythonOperator, PythonSensor]:
                 if (
                     not task_params.get("python_callable")
                     and not task_params.get("python_callable_name")
                     and not task_params.get("python_callable_file")
                 ):
                     raise Exception(
```
Comment (ajbosco) [inline dagfactory/dagbuilder.py:204]:
```suggestion
                    # pylint: disable=line-too-long
                    raise Exception(
```

**Review by ajbosco — APPROVED** [review]

File: dagfactory/dagbuilder.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
 from airflow.models import BaseOperator
 from airflow.operators.python_operator import PythonOperator, BranchPythonOperator
+from airflow.sensors.python import PythonSensor
```
Comment (ajbosco) [inline dagfactory/dagbuilder.py:13]: Can you move this to line 42? `PythonSensor` wasn't introduced until later and I'd like to maintain backwards compatibility with older Airflow versions if possible

Comment (arnekaiser-otto) [reply dagfactory/dagbuilder.py:13]: Yes sure, sorry for number of tries...

Comment (arnekaiser-otto) [reply dagfactory/dagbuilder.py:13]: Another question with a similar background, I face the callable Problem with the google pubsub sensor as well, the sensor has a python class for minimal processing of the messages as default or you can call an own function, both does not work out of the box, do you have an idea how to solve that?

Comment (ajbosco) [inline dagfactory/dagbuilder.py:13]:
```suggestion
```

**Pull request merged**